config/muxcable.py

```diff
@@ -276,28 +276,34 @@ def hwmode():
 def state(state, port):
     """Show muxcable summary information"""
     if port is not None and port != "all":
-        click.confirm(('Do you wish to change the port {} to {} state, Continue?'.format(port, state)), abort=True)
+        click.confirm(('Do you wish to change the muxcable port {} to {} state, Continue?'.format(port, state)), abort=True)
```

Comment:
Suggest rephrasing message: `'Muxcable at port {} will be changed to {} state. Continue?'

Reply:
fixed